### ISSUE TITLE
Replace deprecated .URL parameter

### ISSUE THREAD

**Issue by victoriadrake** [victoriadrake/hugo-theme-introduction#156]
Running Hugo v0.55.6 I see:

```
WARN 2019/07/21 19:36:38 Page's .URL is deprecated and will be removed in a future release. Use .Permalink or .RelPermalink. If what you want is the front matter URL value, use .Params.url.
```

This needs to be updated in the theme's layout files.

**Comment by tanohzana** [comment]
I can do that if you want ! 😉👌🏽

**Comment by hanzei** [comment]
Thanks @tanohzana :+1:

**Comment by tanohzana** [comment]
Little update on this. I have not forgotten about it. I'll do it today 👌

**Comment by tanohzana** [comment]
@hanzei can reproduce the `warning` 🤔 also I found this online:

https://github.com/gohugoio/hugo/issues/5868

TLDR: Apparently `.URL` is not deprecated for menu. You probably got the warning from something you coded that does not belong to the theme. Check if you have any `.Page.URL`...

Let me know 😉

**Comment by hanzei** [comment]
I'm using `v0.56.1` and I'm not seeing this warning. I also don't see a use of `.URL` in the code outside of menu elements.

@tanohzana @victoriadrake Can you please check again?

**Comment by victoriadrake** [comment]
Can no longer reproduce. :woman_shrugging:
